app/views/templates/Layout.scala.html

```diff
@@ -51,7 +51,8 @@
   additionalScripts: Option[Html] = None,
   showBackLink: Boolean = true,
   hasErrors: Boolean = false,
-  showSignOutOverride: Option[Boolean] = None
+  showSignOutOverride: Option[Boolean] = None,
+  taxRegime: TaxRegime = TaxRegime.Epaye
```

Comment:
a default argument here for `taxRegime` seems a bit dangerous - there's a way to get the taxRegime without an extra parameter here if you look at `serviceName` below

Reply:
I did this to cater for when users haven't started a journey yet and navigate directly to it, and used the default to not have to change all of the Paye pages, then I thought when we introduce the "what regime do you want to pay" page we could remove it.

Reply:
for that case maybe we can have a temporary message like `Set up a payment plan` - so we had this previously:
```scala
  case r: AuthenticatedJourneyRequest[_] => Messages.ServicePhase.serviceName(r.journey.taxRegime).show
  case _ => Messages.ServicePhase.serviceName(TaxRegime.Epaye).show
```
Perhaps this could be:
:
```scala
  case r: AuthenticatedJourneyRequest[_] => Messages.ServicePhase.serviceName(r.journey.taxRegime).show
  case _ => Messages.ServicePhase.`Set up a payment plan`
```
with a TODO there perhaps to double check what that default case should be

Reply:
Ah yeah that's better to be fair, I'll implement